<launch>
    <node pkg="easy_aruco" type="easy_aruco_node" name="charuco_node" output="screen" >
        <param name="object_type" value="charuco_board" />

        <param name="camera_namespace" value="/polarization" />
        <param name="camera_info_topic" value="/polarization/camera_info" />
        <param name="camera_image_topic" value="/polarization/image_raw" />
        <param name="dictionary" value="DICT_6X6_250" />

        <param name="camera_frame" value="pol" />
        <param name="reference_frame" value="pol" />

        <param name="marker_size" value="0.016" />
        <param name="square_size" value="0.024" />
        <param name="square_number_x" value="7" />
        <param name="square_number_y" value="9" />
    </node>
</launch>
